appcues/src/main/java/com/appcues/ui/AppcuesViewModelFactory.kt

```diff
@@ -1,16 +0,0 @@
-package com.appcues.ui
```

Comment:
file was not being used since we are not using Activity anymore.

On second thought, I would be interesting to investigate if we really dont need it or if we should instead find a way to use this factory while creating a custom view, as far as I know this ensures that the viewModel can be recreated with the same dependencies.  (did not look into it)